templates_ark/example_pixel_clustering.ipynb

```diff
@@ -490,42 +490,13 @@
   },
```

Comment:
<p>I don't think we need to keep this separate cell of the notebook. The users can either specify a small list of FOVs, to mirror the functionality of the original code, or all of the FOVs, in which case this one would work. I think this can just replace the stuff above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/643/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>